services/comm/app/models/message.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Message < Comm::ApplicationRecord
-  belongs_to :provider
+  belongs_to :provider, optional: true
```

Comment:
This has more implications.
Look at `message_send.rb` we need a provider to send the message. Its simply that we should have a way to fallback to the platform configuration settings